memgpt/persistence_manager.py

```diff
@@ -71,8 +71,18 @@ def init(self, agent):
 
     def json_to_message(self, message_json) -> Message:
         """Convert agent message JSON into Message object"""
-        timestamp = message_json["timestamp"]
-        message = message_json["message"]
+
+        # get message
+        if "message" in message_json:
+            message = message_json["message"]

```

Comment:
@sarahwooders In which cases do we need to do this unpacking, and when do we not need the unpacking?

I'm guessing the unpacking is due to: https://github.com/cpacker/MemGPT/blob/main/memgpt/persistence_manager.py#L112?

Reply:
I'm actually not sure -- I haven't had this issue before but had to do this to get the tests to pass